src/main/java/uk/gov/hmcts/darts/task/runner/impl/CleanupArmResponseFilesAutomatedTask.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.hmcts.darts.task.runner.impl;
+
+import lombok.extern.slf4j.Slf4j;
+import net.javacrumbs.shedlock.core.LockProvider;
+import uk.gov.hmcts.darts.arm.service.CleanupArmResponseFilesService;
+import uk.gov.hmcts.darts.common.repository.AutomatedTaskRepository;
+import uk.gov.hmcts.darts.task.config.AutomatedTaskConfigurationProperties;
+
+import static uk.gov.hmcts.darts.task.runner.AutomatedTaskName.CLEANUP_ARM_RESPONSE_FILES_TASK_NAME;
+
+@Slf4j
+public class CleanupArmResponseFilesAutomatedTask extends AbstractLockableAutomatedTask {
```

Comment:
general observation - we have loads of these automated task classes now and they all do the same thing - only difference is the task name and the service class. just wondering if we could do this some other way?

Reply:
Maybe best to raise a ticket about this

Reply:
will forward to Hemanta to consider

Reply:
@hemantasharma1129 - might or might not be worth looking into this